saltstack/base/salt/postgresql/init.sls

```diff
@@ -20,6 +28,121 @@ install-postgres:
       - postgresql10-docs
       - postgresql10-devel
 
+
+install-centos-sclo-rh:
+  cmd.run:
+    - name: yum -y install centos-release-scl-rh
```

Comment:
I think this should be replaced with pkg.installed

Reply:
right, let me try it out first